python/ql/src/Summary/LinesOfCode.ql

```diff
@@ -0,0 +1,13 @@
+/**
+ * @id py/summary/lines-of-code
+ * @name Total lines of Python code in the database
+ * @description The total number of lines of Python code across all files, including
+ *   external libraries and auto-generated files. This is a useful metric of the size of a
+ *   database. This query counts the lines of code, excluding whitespace or comments.
+ * @kind metric
+ * @tags summary
+ */
```

Comment:
```suggestion
/**
 * @name Total lines of Python code in the database
 * @description The total number of lines of Python code across all files, including
 *   external libraries and auto-generated files. This is a useful metric of the size of a
 *   database. This query counts the lines of code, excluding whitespace or comments.
 * @kind metric
 * @tags summary
 * @id py/summary/lines-of-code
 */
```
Oh, what a terrible diff this produced. All I did was move the first line to the end... :facepalm:

Reply:
I did consider doing this as well, since it looked strange after I had copied the files :stuck_out_tongue: